flask_swagger.py

```diff
@@ -172,16 +174,14 @@ def spec():
         methods = dict()
         for verb in rule.methods.difference(ignore_verbs):
             verb = verb.lower()
-            if hasattr(endpoint, 'methods') \
-                    and verb in map(lambda m: m.lower(), endpoint.methods) \
-                    and hasattr(endpoint.view_class, verb):
+            if hasattr(endpoint, 'methods') and verb in endpoint.methods:
```

Comment:
1. Why remove the `lower()` are you sure that endpoint.methods is lower case?
2. By not checking `hasattr` and just allowing `getattr` to return `None` you are potentially introducing a `None` value in the methods dict. Is that on purpose?

Reply:
I'm 99% sure there was a bug in this part, so I rewrote it, but I don't remember what kind of bug it was and how to reproduce it. It was year ago after all :P

Reply:
To me it looks more like we are introducing a bug. 
If you remove this change I will approve the pr, this is completely unrelated to what the PR is trying to achieve anyways.

Reply:
Yeah, I will remove this part anyway